### ISSUE TITLE
Additional internal and side-loaded Chrome extensions

### ISSUE THREAD

**Issue by theopolis** [osquery/osquery#2807]
I see at least the internal PDF reader as a Chrome extension on my Linux machine, that is not reported in the `chrome_extensions` table. Side-loaded, or developer extensions, are not included either.

**Comment by muffins** [comment]
@obelisk did you ever get a chance to look into this?

**Comment by obelisk** [comment]
I did a cursory look but didn't dive too far into it. I unfortunately don't have any progress sitting in an old branch either it seems :(

**Comment by rcipkins** [comment]
I just tested this on my Linux machine. It seems that the internal PDF reader is no longer stored as a Chrome extension. I installed a few side-loaded/developer extensions as well and they displayed perfectly fine in the table. Perhaps this was just an issue with an older version of Chrome?

**Comment by mike-myers-tob** [comment]
I confirmed this as well. A while back, the Chrome team deprecated NPAPI plugins, and in so doing, moved the PDF reader to a built-in component of the browser. It's still possible to disable it in a setting, but maybe someone should open a new issue if they want a table that enumerates Chrome settings.

I also confirmed what Rachel saw, that osquery is able to enumerate side-loaded extensions, no problem.
